lms/envs/bok_choy.py

```diff
@@ -1,5 +1,13 @@
 """
-Settings for bok choy tests
+Settings for Bok Choy tests that are used for running CMS and LMS.
```

Comment:
Thanks for spotting this. I wrote it in a generic way, but this particular file is only used in LMS (and likewise for the CMS version). I've updated the docstring to be more specific.